### PR TITLE
add field `label` to imagemap actions

### PR DESCRIPTION
**Pull request by TonPC64**
https://developers.line.biz/en/reference/messaging-api/#imagemap-action-objects

### CODE REVIEW

File: linebot/imagemap.go [hunk 1]
```diff
@@ -71,31 +72,36 @@ type URIImagemapAction struct {
 func (a *URIImagemapAction) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&struct {
 		Type    ImagemapActionType `json:"type"`
+		Label   string             `json:"label"`
```
Comment (oklahomer) [inline linebot/imagemap.go]: The document says this field is optional. Can we include "omitempty" in this tag just like other Actions such as `PostBackAction`, `URIAction`, etc?

File: linebot/imagemap.go [hunk 2]
```diff
@@ -71,31 +72,36 @@ type URIImagemapAction struct {
...
 }
 
 // MarshalJSON method of MessageImagemapAction
 func (a *MessageImagemapAction) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&struct {
-		Type ImagemapActionType `json:"type"`
-		Text string             `json:"text"`
-		Area ImagemapArea       `json:"area"`
+		Type  ImagemapActionType `json:"type"`
+		Label string             `json:"label"`
```
Comment (oklahomer) [inline linebot/imagemap.go]: Same as https://github.com/line/line-bot-sdk-go/pull/176/files#r344946868

**Comment by oklahomer** [comment]
I appreciate your effort to add `go.mod`. However, I believe a single pull request should focus on a single feature/issue. Especially when the new requirements include `google.golang.org/appengine`, which actually seems to be optional dependency to run `examples/echo_bot_handler` with build tag of `-tags appengine`.

**Comment by TonPC64** [comment]
> I appreciate your effort to add `go.mod`. However, I believe a single pull request should focus on a single feature/issue. Especially when the new requirements include `google.golang.org/appengine`, which actually seems to be optional dependency to run `examples/echo_bot_handler` with build tag of `-tags appengine`.
> So could you remove this change from this p-r? Other than that, those changes for labels look good.

ok removed

**Pull request merged**